### ISSUE TITLE
More fields in past_values for moves

### ISSUE THREAD

**Issue by BlackGauna** [PokeAPI/pokeapi#967]
Hey,
first of all, I am not sure if this is a bug or simply not implemented yet.

When looking at the data for moves, I saw that some move changes are not represented in the API.
If a move has changed behavior for stat changes or ailment chance, it still has a past_values entry but with nothing meaningful inside.

Examples:
1. [Fire Blast](https://pokeapi.co/api/v2/move/fire-blast): 
The move's burn chance has changed from 30% to 10% after gold-silver. There is a past_values entry for gold-silver, but the ailment_chance change is not shown. All other values inside the entry are null, so it would seem, nothing is changed at all.

2. [String Shot](https://pokeapi.co/api/v2/move/81/):
String Shot's status effect has changed since gen VI from speed -1 to speed -2. Here is also a past_values entry, but with no meaningful changes listed. The entry is for sun-moon instead of x-y, but that seems to be an issue with the [veekun entry](https://veekun.com/dex/moves/string%20shot#history) that is wrong.

So in summary there should also be fields for meta, stat_changes and effect_entries inside a past_values entry to also reflect changes that are not just power, accuracy, pp or type. 
Maybe the effect_changes field was intended to be used for this purpose, but is currently always empty (at least in the few examples I have checked). However, it would be better to just list all the changes in the past_values entry anyway, imo.

**Comment by Naramsim** [comment]
We lack this data, and most of the past values
